### ISSUE TITLE
give user control of sex-ratio multiplier for 1-sex models

### ISSUE THREAD

**Issue by iantaylor-NOAA** [r4ss/r4ss#248]
Max Cardinale pointed out in a VLab forums post that the r4ss time series plots for 1-sex model divide spawning biomass by 2 to represent female-only spawning biomass. However, some users may want control over this.
In the function `SSplotTimeseries`, there is an input `bioscale` which by default is replaced by 0.5 for 1-sex models and 1.0 for 2-sex models. However, it would be easier if this input were available in `SS_plots` which would pass it to `SSplotTimeseries`. For models with spawning output represented in big numbers, like millions of eggs, `bioscale` could theoretically be used to rescale the units in the vertical axis to easier-to-read values. The axis label reads "Spawning output" for models where spawning output isn't proportional to biomass for mature fish, so that label could remain as is.

In `SSplotComparisons`, the division by 2 is hardwired. It would be useful to add a new input, perhaps also named `bioscale`, to give users control of this multiplier. In this function, large values for spawning output are automatically scaled down and the axis label adjusted to be note that the units are in 1000s, millions, or billions, so you would never want to apply a bioscale value other than 1.0 or 0.5.

**Comment by taylori** [comment]
Rick has said that the next version of SS will give the user the choice whether or not to multiply the spawning biomass by the sex ratio in 1-sex model. The r4ss functions can then be modified to detect that this option is being used and just use the output from SS without modification. I'll leave this issue open until a new version of SS is out to remember to make that change.

**Comment by iantaylor-NOAA** [comment]
The option to have a 1-sex model in SS with SSB automatically multiplied by the fraction females has been available in SS for a while now but r4ss still isn't adapted to it.

Steps required are
- [ ] check for users having input `-1 #_Ngenders: 1, 2, -1  (use -1 for 1 sex setup with SSB multiplied by female_frac parameter)`
- [ ] either take out `bioscale` input from `SSplotTimeseries` or change to default to 1 for Ngenders = -1.
- [ ] take away division by 2 in `SSplotComparisons` for models with Ngenders = -1

**Comment by iantaylor-NOAA** [comment]
With commit f07af64, I removed the automatic scaling of biomass in both `SSplotTimeseries` and `SSplotComparisons`. Now if you want to compare the scale of a 1-sex model to a 2-sex model, the best practice would be to use the new option in SS:
```
-1 #_Ngenders: 1, 2, -1 (use -1 for 1 sex setup with SSB multiplied by female_frac parameter)
```
There doesn't yet seem to be any easy way to tell from the Report file whether the -1 or 1 option was used in option was used, so I didn't try to automatically scale based on the user's choice.

**Comment by iantaylor-NOAA** [comment]
I failed to change the section in `SSplotComparisons` which related to MCMC models when removing the automatic division by 2 in 1-sex models. Commit 7dae19dc31fcc2a7435c2a9f7707416a493fc915 corrects that and also removes division by 2 in `SStableComparisons`. I think this issue is now complete.

**Comment by iantaylor-NOAA** [comment]
Thanks to @akatan999 for reporting a scaling issue in the stock-recruit plot that I traced back to this same issue.
There remained a scaling of 0.5 in place for 1-sex models in the `SSplotSpawnrecruit()` function. It's now fixed in the main branch via commit b250371. 

If you want to adjust spawning biomass by the sex ratio, use Nsexes = -1 in the data file: 
`-1 #_Nsexes: 1, 2, -1  (use -1 for 1 sex setup with SSB multiplied by female_frac parameter)` 
This will make it happen within SS3 which is better than the old way of applying the ratio in r4ss.
